source/Calamari.CloudAccounts/Azure/AccountVariables.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
```

Comment:
Should these files be left in Sashimi and only keep what you need in Calamari.Azure

Reply:
Yep, I've reverted them and only brought over what is required.